independent-publisher-connectors/Edatalia sign online/apiDefinition.swagger.json

```diff
@@ -0,0 +1,625 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "edatalia Sign Online",
+    "description": "Connector to a Sign service to create online signatures to customers",
+    "termsOfService": "https://firmar.online/condiciones-generales-de-contratacion",
+    "contact": {
+      "name": "Axazure",
+      "url": "https://www.axazure.com"
+    },
+    "version": "v40"
+  },
+  "host": "restapi.firmar.info",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/PSC/v40/DocumentSet/Simple": {
+      "post": {
+        "tags": [
+          "DocumentSet"
+        ],
+        "summary": "Send simple document to sign",
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@marpedroche Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**